ploneorg/theme/theme/plone-open-garden.html

```diff
@@ -2084,15 +2084,15 @@
     <h1 class="hidden-phone">Live out the PLOG community.</h1>
     <h1 class="visible-phone">Be PLOG.</h1>
 	<p class="lead">
-        PLOG is the Italian Plone event taking place in Sorrento <a href="https://plone.org/events/sprints/sorrento-sprint-on-frontend-modernization">April 12-19, 2019</a>, aimed at those who are passionate
+        PLOG is the Italian Plone event taking place in Sorrento, aimed at those who are passionate
```

Comment:
Maybe we could keep the event URL the same from year to year, but, each year, clone it to create the past year's event (at a URL that includes the year) and then update the event. This would keep the relevant URL constant and would not require you to update this page.

Reply:
This idea I'm not so enthusiastic about as keeping the registration form the same. It sounds like more trouble than just editing the URL on this page - and we are going to have to edit this page each year to update the dates. Let me know if you feel strongly about this.

Reply:
I don't feel strongly about this but keeping the event URL the same year after year probably will improve Google fu, no?

Reply:
I suppose having a standard PLOG event URL would improve its analytics, although it's probably more important to worry about the analytics of this page. I made a note to do what you've suggested next year. 

Regardless of that I am removing the event link here because the blue link on the dark background is illegible.